src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/HistoricalBeneficialOwnerValidator.java

```diff
@@ -27,8 +27,11 @@ public Errors validate(List<TrustDataDto> trustDataDtoList, Errors errors, Strin
             List<HistoricalBeneficialOwnerDto> historicalBeneficialOwners = trustDataDto
                     .getHistoricalBeneficialOwners();
             if (!CollectionUtils.isEmpty(historicalBeneficialOwners)) {
+                boolean isUnableToObtainAllTrustInfo = trustDataDto.getUnableToObtainAllTrustInfo();
                 for (HistoricalBeneficialOwnerDto historicalBeneficialOwnerDto : historicalBeneficialOwners) {
-                    validateCeasedDate(historicalBeneficialOwnerDto.getCeasedDate(), errors, loggingContext);
+                    if (!isUnableToObtainAllTrustInfo){
```

Comment:
Minor, but space required before the opening curly brace.

Reply:
Formatted code